app/views/my_modules/protocols.html.erb

```diff
@@ -41,7 +41,20 @@
         <span class="task-section-title">
           <h2><%= t('my_modules.assigned_items.title') %></h2>
         </span>
-    </a>
+      </a>
+      <div class="actions-block">
+        <div class="dropdown repositories-assign-container"
+             data-repositories-url="<%= dropdown_list_my_module_my_module_repositories_path(@my_module) %>"
+        >
+          <a href="#" id="repositories-assign-button" class="btn btn-secondary btn-block" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            <span class="fas fa-file-signature"></span>
+            <span><%= t('my_modules.assigned_items.assign_from') %></span>
+            <span class="caret pull-right"></span>
+          </a>
+          <div class="dropdown-menu repositories-dropdown-menu" aria-labelledby="repositories-assign-button">
```

Comment:
Why don't you use `ul` and `li` elements? (Bootstrap standard)
The hover style is missing.

Reply:
Fixed